apps/mobile/src/components/browser/browser-sheet/tabs/suggested.tsx

```diff
@@ -0,0 +1,92 @@
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { BottomSheetScrollView } from '@gorhom/bottom-sheet';
+import { t } from '@lingui/macro';
+import { Image } from 'expo-image';
+
+import { Box, Text } from '@leather.io/ui/native';
+
+import { DappCard } from '../dapp-card';
+import { URL_SEARCH_HEIGHT } from '../utils';
+
+interface SuggestedProps {
+  searchUrl(url: string): void;
+}
+
+export function Suggested({ searchUrl }: SuggestedProps) {
+  const { bottom } = useSafeAreaInsets();
+  return (
+    <BottomSheetScrollView contentInset={{ bottom: bottom + URL_SEARCH_HEIGHT }}>
```

Comment:
I would love to see use of Layout components here. Where `BrowserSuggestedTab` adds content & functionality to `BrowserSuggestedTabLayout` that encapsulates all the styles